gapis/api/vulkan/vulkan_terminator.go

```diff
@@ -312,7 +312,7 @@ func cutCommandBuffer(ctx context.Context, id api.CmdID,
 	// idx[3] is the secondary command buffer index inside a vkCmdExecuteCommands
 	// idx[4] is the secondary command inside the secondary command-buffer
 	submitCopy := cb.VkQueueSubmit(a.Queue, a.SubmitCount, a.PSubmits, a.Fence, a.Result)
-	submitCopy.Extras().Add(a.Extras().All()...)
+	submitCopy.Extras().CloneFromOrPanic(a.Extras().All()...)
```

Comment:
The only reason is that we use Add(a.Extras().All()....) A lot elsewhere, when we are not cloning an atom.